cmd/azure.go

```diff
@@ -29,6 +29,7 @@ import (
 	"net/url"
 	"strconv"
```

Comment:
yes, but there is no other choice, ab suggested this solution and say that multipart is not supported if server is restarted.
what can be done is store the metadata on a tmp `uuid` object and use it during complete multipart, will check with ab if that can be done. if you can think of any other way let me know.